src/tests/admin/conferenceroomoutoforderplanning/IfOutOfOrderIsCreatedInTheFutureCalendarIconIsDisplayed.java

```diff
@@ -0,0 +1,59 @@
+package tests.admin.conferenceroomoutoforderplanning;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomOutOfOrderPlanningPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC04: Verify that Calendar icon appears in out of order column when the schedule to be 
+ * out of order is not yet in the time
+ * @author Yesica Acha
+ *
+ */
+public class IfOutOfOrderIsCreatedInTheFutureCalendarIconIsDisplayed {
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> testData = excelReader.getMapValues("OutOfOrderPlanning");
+	String roomName = testData.get(2).get("Room Name");
```

Comment:
You are using only "roomName" and "title" variables as global, you have to declare others (startDate, endDate, startTime, etc) inside the test method